hkmc2/shared/src/test/mlscript/backlog/ToTriage.mls

```diff
@@ -352,8 +352,11 @@ mkStr of ... // hello
 
 data class Foo(x, y)
 
-// TODO missing sanity check
+:e
 Foo(1)
+//│ ╔══[ERROR] Expected 2 arguments, got 1
+//│ ║  l.356: 	Foo(1)
+//│ ╙──       	   ^^^
```

Comment:
The intent of this test was to show the lack of runtime sanity checks. So you should revert it to something like
```suggestion
// TODO missing sanity check
:re
id(Foo)(1)
```